pkg/asset/installconfig/kubevirt/validation.go

```diff
@@ -0,0 +1,119 @@
+package kubevirt
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net"
+
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/kubevirt"
+	"github.com/openshift/installer/pkg/types/kubevirt/validation"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+// Validate executes kubevirt specific validation
+func Validate(ic *types.InstallConfig, clientBuilder ClientBuilderFuncType) error {
+	kubevirtPlatformPath := field.NewPath("platform", "kubevirt")
+
+	if ic.Platform.Kubevirt == nil {
+		return errors.New(field.Required(
+			kubevirtPlatformPath,
+			"validation requires a Engine platform configuration").Error())
```

Comment:
Engine is an ovirt term, this comment needs rewording.

Reply:
Fixed